submissions/aymkin/js-dom/index.js

```diff
@@ -0,0 +1,110 @@
+    description:
+      'The pear (/ˈpɛər/) tree and shrub are a species of genus Pyrus /ˈpaɪrəs/, in the family Rosaceae, bearing the pomaceous fruit of the same name. Several species of pear are valued for their edible fruit and juices while others are cultivated as trees.'
+  },
+  {
+    name: 'Durian',
+    imgSrc: './img/durian.png',
+    description:
+      'The durian (/ˈdjʊəriən, ˈdʊr-, -æn/)[2] is the fruit of several tree species belonging to the genus Durio. There are 30 recognised Durio species, at least nine of which produce edible fruit, with over 100 named varieties in Indonesia,[3] 300 in Thailand and 100 in Malaysia.[4] Durio zibethinus is the only species available in the international market: other species are sold in their local regions.'
+  }
+];
+
+const doc = document;
+
+const container = doc.createElement('div');
+container.className = 'container';
+doc.body.appendChild(container);
+
+const pageName = doc.createElement('h1');
+pageName.className = 'page-name';
+pageName.textContent = 'JS DOM';
+container.appendChild(pageName);
+
+const dictionaryName = doc.createElement('h2');
+dictionaryName.className = 'dictionary-name';
+dictionaryName.textContent = 'Fruit dictionary';
+container.appendChild(dictionaryName);
+
+const divMain = doc.createElement('div');
+divMain.className = 'main';
+container.appendChild(divMain);
+
+const divMenu = doc.createElement('div');
+divMenu.className = 'menu';
+divMain.appendChild(divMenu);
+
+const navigation = doc.createElement('ul');
+navigation.className = 'navigation';
+const nav = doc.querySelectorAll('li');
+divMenu.appendChild(navigation);
+
+fruits.forEach((val, key) => {
+  const li = doc.createElement('li');
+  li.className = 'list-element';
+  li.textContent = fruits[key].name;
+  if (key === 0) {
+    li.className = 'list-element clicked';
+  }
+  navigation.appendChild(li);
+});
+
+const divImage = doc.createElement('div');
+divImage.className = 'image';
+divMain.appendChild(divImage);
+
+const image = doc.createElement('img');
+image.className = 'image';
+image.setAttribute('src', fruits[0].imgSrc);
+divImage.appendChild(image);
+
+const divDescription = doc.createElement('div');
+divDescription.className = 'description';
+divMain.appendChild(divDescription);
+
+const descriptionHeader = doc.createElement('h3');
+descriptionHeader.textContent = fruits[0].name;
+divDescription.appendChild(descriptionHeader);
+
+const descriptionParah = doc.createElement('p');
+descriptionParah.textContent = fruits[0].description;
+divDescription.appendChild(descriptionParah);
+const eventTarget = doc.querySelector('ul');
+
+eventTarget.addEventListener('click', evt => {
+  const lis = doc.querySelectorAll('li');
+  // onckick reset styles for all li elements
+  lis.forEach(element => {
+    element.className = 'list-element';
+  });
+
+  fruits.forEach((val, key) => {
+    if (evt.target.textContent === fruits[key].name) {
+      image.setAttribute('src', fruits[key].imgSrc);
```

Comment:
You can use `image.src = fruits[key.imgSrc]` it will be easy to read
It is unnecessary to correct it but take into account😉

Reply:
done, I have corrected that

Reply:
done